src/doc/rustc/src/platform-support.md

```diff
@@ -205,6 +205,7 @@ target | std | host | notes
 `powerpc64-wrs-vxworks` | ? |  |
 `powerpc64le-unknown-linux-musl` | ? |  |
 `riscv32gc-unknown-linux-gnu` |   |   | RISC-V Linux (kernel 5.4, glibc 2.33)
+`s390x-unknown-linux-musl` | ✓ | ✓ | S390x Linux (kernel 2.6.32, MUSL)
```

Comment:
This doesn't seem right - don't you need to update libc before you can bootstrap on this target?

> host: A ✓ indicates that rustc and cargo can run on the host platform.

Reply:
Right now since the `std` does not build for the target, and consequentially the compiler cannot be produced for the target either, I think these checkboxes should be removed in this PR.